modules/ROOT/pages/database-internals/store-formats.adoc

```diff
@@ -11,6 +11,9 @@ The available formats are:
 
 Neo4j's newest store format is `block`.
 It has the best performance and supports the highest limits.
+For example, compared to the existing record format, the `block` format has roughly 40% better performance when the whole graph is in memory.
+That lead increases to 70% when only a third of the graph can fit in memory.
```

Comment:
I don't think we should have things like performance numbers in official documentation. It is not a guarantee and is highly data dependent, some users may see much better results, and some no increase at all

Reply:
Users want more specific information about the differences and which format to use and when, but I agree that these percentages may change.